netstandard/ref/mscorlib.appdomain.cs

```diff
@@ -148,8 +148,8 @@ public sealed partial class AppDomain : System.MarshalByRefObject
         [System.ObsoleteAttribute("AppDomain.GetCurrentThreadId has been deprecated because it does not provide a stable Id when managed threads are running on fibers (aka lightweight threads). To get a stable identifier for a managed thread, use the ManagedThreadId property on Thread.  http://go.microsoft.com/fwlink/?linkid=14202", false)]
         public static int GetCurrentThreadId() { throw null; }
         public object GetData(string name) { throw null; }
-        public new System.Type GetType() { throw null; }
-        public override object InitializeLifetimeService() { throw null; }
+//NOOP       public new System.Type GetType() { throw null; }
```

Comment:
What is the problem with keeping these in the standard? .NET Core can be compatible without even doing anything for these, as not all overrides need to be present in the implementation to be compatible.